### PR TITLE
<Description> : Adding uppercase and lower case for character extensions.

### PR DESCRIPTION
**Pull request by Khalian**
## Checklist

- [x] New Extension
- [x] New Test
- [ ] Changed more than one extension, but all changes are related
- [ ] Trivial change (doesn't require changelog)

### CODE REVIEW

**Comment by lfarah** [comment]
I think it's better if we don't camel case ```lowerCase``` and ```upperCase``` since Apple already uses it lowercased (no puns intended 😂): ```lowercased()``` and ```uppercased()```

**Comment by Khalian** [comment]
Done.

File: EZSwiftExtensionsTests/CharacterTests.swift [hunk 1]
```diff
@@ -30,4 +31,14 @@ class CharacterTests: XCTestCase {
         XCTAssertEqual(digitChr.toInt, int)
         XCTAssertEqual(pchr.toInt, nil)
     }
+    
+    func testUpperCase() {
```
Comment (lfarah) [inline EZSwiftExtensionsTests/CharacterTests.swift]: Should change name for tests too

File: EZSwiftExtensionsTests/CharacterTests.swift [hunk 2]
```diff
@@ -30,4 +31,14 @@ class CharacterTests: XCTestCase {
...
+        XCTAssertEqual(pchr.uppercased, pchrUpper)
+        XCTAssertEqual(digitChr.uppercased, digitChr)
+    }
+    
+    func testLowerCase() {
```
Comment (lfarah) [inline EZSwiftExtensionsTests/CharacterTests.swift]: Same here

Comment (Khalian) [inline EZSwiftExtensionsTests/CharacterTests.swift]: Done.

**Comment by piv199** [comment]
`String().lowercased()` but `Character().lowercased`, why?
Also, doesn't String and Character have common protocol to extend with this function?

**Pull request merged**